openslides/motions/static/js/motions/base.js

```diff
@@ -288,6 +289,37 @@ angular.module('OpenSlidesApp.motions', [
                 getReason: function (versionId) {
                     return this.getVersion(versionId).reason;
                 },
+                // full state name - optional with custom state name extension
+                // depended by state and provided by a custom comment field
+                getStateName: function () {
+                    var additionalName = '';
+                    if (this.state.show_state_extension_field) {
+                        // check motion comment fields for flag 'forState'
+                        var fields = Config.get('motions_comments').value;
+                        for (var i = 0; i < fields.length; i++) {
+                            if (fields[i].forState) {
+                                additionalName = ' ' + this.comments[i];
```

Comment:
You should break the loop if the first entry was found. Better use [_.findIndex()](https://lodash.com/docs/4.16.4#findIndex).